lib/services/common-api-presenter.js

```diff
@@ -209,28 +211,24 @@ function CommonApiPresenterFactory(
             var scope = self.res.locals.scope;
 
             var promises = [
-                lookupService.ipAddressToMacAddress(self.res.locals.ipAddress),
-                templates.get(name, scope)
+                Promise.props({
+                    server: configuration.get('apiServerAddress', '10.1.1.1'),
+                    port: configuration.get('apiServerPort', 80),
+                    ipaddress: self.res.locals.ipAddress,
+                    netmask: configuration.get('dhcpSubnetMask', '255.255.255.0'),
+                    gateway: configuration.get('dhcpGateway', '10.1.1.1'),
+                    macaddress: lookupService.ipAddressToMacAddress(self.res.locals.ipAddress),
+                    sku: Env.get('config', {}, [ scope[0] ]),
```

Comment:
What the `'config'` here means? Only reads those configuration entries from sku pack's "skuConfig" definition? User may confuse it is from RackHD configuration file (monorail.json).  

Meanwhile, why only use the scope[0]? It is the SKU scope? I think it's better to have a comment here.